pkg/core/xds/types.go

```diff
@@ -118,6 +125,17 @@ func (e Endpoint) IsExternalService() bool {
 	return e.ExternalService != nil
 }
 
+func (e Endpoint) LocalityString() string {
+	if e.Locality == nil {
+		return ""
+	}
+	return e.Locality.Region + e.Locality.Zone + e.Locality.SubZone
```

Comment:
we need a separator between them (maybe `/` ?). Otherwise
```
Region: "abc"
Zone: ""
Subzone: ""
```
is the same as
```
Region: "a"
Zone: "b"
Subzone: "c"
```

Reply:
Good catch!